mail/dovecot2/Portfile

```diff
@@ -79,310 +146,312 @@ configure.args      --sysconfdir=${prefix}/etc \
+        configure.args-append       --with-mysql
+    }
 
-variant mysql51 \
-    conflicts mysql5 mysql55 mysql56 mariadb percona \
-    description "Enable MySQL 5.1 support" {
-    depends_lib-append          port:mysql51
-    configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql51/bin/mysql_config
-    configure.args-append       --with-mysql
-}
+    variant mysql51 \
+        conflicts mysql5 mysql55 mysql56 mariadb percona \
+        description "Enable MySQL 5.1 support" {
+        depends_lib-append          port:mysql51
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql51/bin/mysql_config
+        configure.args-append       --with-mysql
+    }
 
-variant mysql5 \
-    conflicts mysql51 mysql55 mysql56 mariadb percona \
-    description "Enable MySQL 5.1 support" {
-    depends_lib-append          port:mysql5
-    configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql5/bin/mysql_config
-    configure.args-append       --with-mysql
-}
+    variant mysql5 \
+        conflicts mysql51 mysql55 mysql56 mariadb percona \
+        description "Enable MySQL 5.1 support" {
+        depends_lib-append          port:mysql5
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql5/bin/mysql_config
+        configure.args-append       --with-mysql
+    }
 
-variant no_startupitem description {Do not install a launchd plist} {}
-
-if {![variant_isset "no_startupitem"]} {
-    if {![variant_isset "apns"]} {
-        startupitem.create      yes
-        startupitem.executable  ${prefix}/sbin/${base_name}
-        startupitem.pidfile     auto ${prefix}/var/run/${base_name}/master.pid
-    } else {
-        set daemon_uniquename   \
-                                org.macports.${name}
-        set pushnotify_uniquename \
-                                org.macports.${base_name}-apns
-        startupitem.type        launchd
-        startupitem.create      no
-        # Inject our own launchd plists to include pushnotify.pl
-        startupitems \
-            name                ${name} \
-            location            LaunchDaemons \
-            uniquename          ${daemon_uniquename} \
-            plist               ${daemon_uniquename}.plist \
-            name                ${base_name}-apns \
-            location            LaunchDaemons \
-            uniquename          ${pushnotify_uniquename} \
-            plist               ${pushnotify_uniquename}.plist
-
-        post-destroot {
-            xinstall -m 0755 -d \
-                ${destroot}/Library/LaunchDaemons
-            foreach uniquename "${daemon_uniquename} ${pushnotify_uniquename}" {
-                xinstall -m 644 -W ${filespath} ${uniquename}.plist \
-                    ${destroot}/Library/LaunchDaemons
-                reinplace "s|@PREFIX@|${prefix}|g" \
-                    ${destroot}/Library/LaunchDaemons/${uniquename}.plist
+    variant no_startupitem description {Do not install a launchd plist} {}
```

Comment:
We don't like to use "negative" variant names. Can you please call this variant `startupitem` instead?

Also, does this really even need to be in a variant at all? startupitems don't do anything unless the user calls them. Plenty of ports have launchd plists and don't use variants.

Reply:
FYI, they are normally invoked with:

`sudo port load <portname>
`

To turn it off, you use:

`sudo port unload <portname>
`

Reply:
This variant is pre-existing and apparently old. I don’t have any insight about why it’s there and would be happy to delete it.

Reply:
You're right. Sorry about that. But please fix that in another PR at some point.